Integrations/python/deephaven/java/__init__.py

```diff
@@ -0,0 +1,185 @@
+#
+# Copyright (c) 2016 - 2021 Deephaven Data Labs and Patent pending
+#
+"""
+A deephaven submodule for accessing java types from Python
+"""
+
+import jpy
+import wrapt
+import sys
+
+# None until the first _defineSymbols() call
+_Runtime_ = None
+
+#
+# Define Java types
+# Java types from Python are represented as jpy wrappers for the corresponding
+# Java type object.
+# 
+
+# None until the first _defineSymbols() call
+# primitives
+boolean = None
+byte = None
+char = None
+double = None
+float = None
+float32 = None
+float64 = None
+int = None
+int16 = None
+int32 = None
+int64 = None
+long = None
+short = None
+# java.io
+File = None
+# java.lang
+Boolean = None
+Byte = None
+Character = None
+Double = None
+Float = None
+Integer = None
+Long = None
+Math = None
+Short = None
+String = None
+System = None
+# java.math
+BigDecimal = None
+BigInteger = None
+# java.nio
+Buffer = None
+ByteBuffer = None
+CharBuffer = None
+DoubleBuffer = None
+FloatBuffer = None
+IntBuffer = None
+LongBuffer = None
+ShortBuffer = None
+# java.text
```

Comment:
I'm not opposed to this view.  I'm also not opposed to having this contain all of the raw Java types.

Reply:
I'm not sure I'm a big fan of re-defining the java namespace hierarchy to make things easier in a large generic sense. I'm happy to have something like `java.lang.*` imported by default, and I'm ok to define primitives and DH types as top level, but redefining a lot of java packages to make it "easier" is a bit subjective IMO.

So, another possibility, is to have a dynamic python module import that maps to jpy types - I've seen other python/java bridges do this (https://www.jython.org/).

```python
from java.text import SimpleDateFormat
```

It could be nice to hook this up to jpy.

Reply:
to be able to use the java namespace directly would be great, not sure if it can be achieved on a CPython-Java bridge other than a JVM-based Python interpreter.

Reply:
I'm pretty confident we could create a python module logic that will "magically" be able to resolve any jpy/java types via python imports.

Reply:
I think Jython's syntax looks nice and would be a good target.  The exact implementation is probably irrelevant since Jython is 100% Java and has no bridge.

Do we know for certain that this is feasible?  How long to accomplish?

Reply:
It looks like Finders are the Python magic to make this feature work.
https://realpython.com/python-import/#finders-and-loaders
https://docs.python.org/3/library/importlib.html
https://www.python.org/dev/peps/pep-0302/
https://www.python.org/dev/peps/pep-0451/